skins/oasis/css/core/ads.scss

```diff
@@ -14,6 +14,10 @@
 	display: none !important;
 }
 
+.source-point .wikia-ad.default-height {
```

Comment:
As we discussed offline having it in JS only doesn't work nice since the end user can still see the jumping. Let's leave it as it is now.